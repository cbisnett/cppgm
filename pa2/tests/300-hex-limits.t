0x7FFFFFFF 0x7FFFFFFF 0x7FFFFFFFl 0x7FFFFFFFl 0x7FFFFFFFL 0x7FFFFFFFL 0x7FFFFFFFll 0x7FFFFFFFll 0x7FFFFFFFLL 0x7FFFFFFFLL 
0x7FFFFFFFu 0x7FFFFFFFu 0x7FFFFFFFul 0x7FFFFFFFlu 0x7FFFFFFFuL 0x7FFFFFFFLu 0x7FFFFFFFull 0x7FFFFFFFllu 0x7FFFFFFFuLL 0x7FFFFFFFLLu 
0x7FFFFFFFU 0x7FFFFFFFU 0x7FFFFFFFUl 0x7FFFFFFFlU 0x7FFFFFFFUL 0x7FFFFFFFLU 0x7FFFFFFFUll 0x7FFFFFFFllU 0x7FFFFFFFULL 0x7FFFFFFFLLU 
0x7FFFFFFF_ud1 0x7FFFFFFF_ud1 0x7FFFFFFF_ud1l 0x7FFFFFFFl_ud1 0x7FFFFFFF_ud1L 0x7FFFFFFFL_ud1 0x7FFFFFFF_ud1ll 0x7FFFFFFFll_ud1 0x7FFFFFFF_ud1LL 0x7FFFFFFFLL_ud1 
0x7FFFFFFE 0x7FFFFFFE 0x7FFFFFFEl 0x7FFFFFFEl 0x7FFFFFFEL 0x7FFFFFFEL 0x7FFFFFFEll 0x7FFFFFFEll 0x7FFFFFFELL 0x7FFFFFFELL 
0x7FFFFFFEu 0x7FFFFFFEu 0x7FFFFFFEul 0x7FFFFFFElu 0x7FFFFFFEuL 0x7FFFFFFELu 0x7FFFFFFEull 0x7FFFFFFEllu 0x7FFFFFFEuLL 0x7FFFFFFELLu 
0x7FFFFFFEU 0x7FFFFFFEU 0x7FFFFFFEUl 0x7FFFFFFElU 0x7FFFFFFEUL 0x7FFFFFFELU 0x7FFFFFFEUll 0x7FFFFFFEllU 0x7FFFFFFEULL 0x7FFFFFFELLU 
0x7FFFFFFE_ud1 0x7FFFFFFE_ud1 0x7FFFFFFE_ud1l 0x7FFFFFFEl_ud1 0x7FFFFFFE_ud1L 0x7FFFFFFEL_ud1 0x7FFFFFFE_ud1ll 0x7FFFFFFEll_ud1 0x7FFFFFFE_ud1LL 0x7FFFFFFELL_ud1 
0x80000000 0x80000000 0x80000000l 0x80000000l 0x80000000L 0x80000000L 0x80000000ll 0x80000000ll 0x80000000LL 0x80000000LL 
0x80000000u 0x80000000u 0x80000000ul 0x80000000lu 0x80000000uL 0x80000000Lu 0x80000000ull 0x80000000llu 0x80000000uLL 0x80000000LLu 
0x80000000U 0x80000000U 0x80000000Ul 0x80000000lU 0x80000000UL 0x80000000LU 0x80000000Ull 0x80000000llU 0x80000000ULL 0x80000000LLU 
0x80000000_ud1 0x80000000_ud1 0x80000000_ud1l 0x80000000l_ud1 0x80000000_ud1L 0x80000000L_ud1 0x80000000_ud1ll 0x80000000ll_ud1 0x80000000_ud1LL 0x80000000LL_ud1 
0xFFFFFFFF 0xFFFFFFFF 0xFFFFFFFFl 0xFFFFFFFFl 0xFFFFFFFFL 0xFFFFFFFFL 0xFFFFFFFFll 0xFFFFFFFFll 0xFFFFFFFFLL 0xFFFFFFFFLL 
0xFFFFFFFFu 0xFFFFFFFFu 0xFFFFFFFFul 0xFFFFFFFFlu 0xFFFFFFFFuL 0xFFFFFFFFLu 0xFFFFFFFFull 0xFFFFFFFFllu 0xFFFFFFFFuLL 0xFFFFFFFFLLu 
0xFFFFFFFFU 0xFFFFFFFFU 0xFFFFFFFFUl 0xFFFFFFFFlU 0xFFFFFFFFUL 0xFFFFFFFFLU 0xFFFFFFFFUll 0xFFFFFFFFllU 0xFFFFFFFFULL 0xFFFFFFFFLLU 
0xFFFFFFFF_ud1 0xFFFFFFFF_ud1 0xFFFFFFFF_ud1l 0xFFFFFFFFl_ud1 0xFFFFFFFF_ud1L 0xFFFFFFFFL_ud1 0xFFFFFFFF_ud1ll 0xFFFFFFFFll_ud1 0xFFFFFFFF_ud1LL 0xFFFFFFFFLL_ud1 
0xFFFFFFFF 0xFFFFFFFF 0xFFFFFFFFl 0xFFFFFFFFl 0xFFFFFFFFL 0xFFFFFFFFL 0xFFFFFFFFll 0xFFFFFFFFll 0xFFFFFFFFLL 0xFFFFFFFFLL 
0xFFFFFFFFu 0xFFFFFFFFu 0xFFFFFFFFul 0xFFFFFFFFlu 0xFFFFFFFFuL 0xFFFFFFFFLu 0xFFFFFFFFull 0xFFFFFFFFllu 0xFFFFFFFFuLL 0xFFFFFFFFLLu 
0xFFFFFFFFU 0xFFFFFFFFU 0xFFFFFFFFUl 0xFFFFFFFFlU 0xFFFFFFFFUL 0xFFFFFFFFLU 0xFFFFFFFFUll 0xFFFFFFFFllU 0xFFFFFFFFULL 0xFFFFFFFFLLU 
0xFFFFFFFF_ud1 0xFFFFFFFF_ud1 0xFFFFFFFF_ud1l 0xFFFFFFFFl_ud1 0xFFFFFFFF_ud1L 0xFFFFFFFFL_ud1 0xFFFFFFFF_ud1ll 0xFFFFFFFFll_ud1 0xFFFFFFFF_ud1LL 0xFFFFFFFFLL_ud1 
0x100000000 0x100000000 0x100000000l 0x100000000l 0x100000000L 0x100000000L 0x100000000ll 0x100000000ll 0x100000000LL 0x100000000LL 
0x100000000u 0x100000000u 0x100000000ul 0x100000000lu 0x100000000uL 0x100000000Lu 0x100000000ull 0x100000000llu 0x100000000uLL 0x100000000LLu 
0x100000000U 0x100000000U 0x100000000Ul 0x100000000lU 0x100000000UL 0x100000000LU 0x100000000Ull 0x100000000llU 0x100000000ULL 0x100000000LLU 
0x100000000_ud1 0x100000000_ud1 0x100000000_ud1l 0x100000000l_ud1 0x100000000_ud1L 0x100000000L_ud1 0x100000000_ud1ll 0x100000000ll_ud1 0x100000000_ud1LL 0x100000000LL_ud1 
0x7FFFFFFFFFFFFFFF 0x7FFFFFFFFFFFFFFF 0x7FFFFFFFFFFFFFFFl 0x7FFFFFFFFFFFFFFFl 0x7FFFFFFFFFFFFFFFL 0x7FFFFFFFFFFFFFFFL 0x7FFFFFFFFFFFFFFFll 0x7FFFFFFFFFFFFFFFll 0x7FFFFFFFFFFFFFFFLL 0x7FFFFFFFFFFFFFFFLL 
0x7FFFFFFFFFFFFFFFu 0x7FFFFFFFFFFFFFFFu 0x7FFFFFFFFFFFFFFFul 0x7FFFFFFFFFFFFFFFlu 0x7FFFFFFFFFFFFFFFuL 0x7FFFFFFFFFFFFFFFLu 0x7FFFFFFFFFFFFFFFull 0x7FFFFFFFFFFFFFFFllu 0x7FFFFFFFFFFFFFFFuLL 0x7FFFFFFFFFFFFFFFLLu 
0x7FFFFFFFFFFFFFFFU 0x7FFFFFFFFFFFFFFFU 0x7FFFFFFFFFFFFFFFUl 0x7FFFFFFFFFFFFFFFlU 0x7FFFFFFFFFFFFFFFUL 0x7FFFFFFFFFFFFFFFLU 0x7FFFFFFFFFFFFFFFUll 0x7FFFFFFFFFFFFFFFllU 0x7FFFFFFFFFFFFFFFULL 0x7FFFFFFFFFFFFFFFLLU 
0x7FFFFFFFFFFFFFFF_ud1 0x7FFFFFFFFFFFFFFF_ud1 0x7FFFFFFFFFFFFFFF_ud1l 0x7FFFFFFFFFFFFFFFl_ud1 0x7FFFFFFFFFFFFFFF_ud1L 0x7FFFFFFFFFFFFFFFL_ud1 0x7FFFFFFFFFFFFFFF_ud1ll 0x7FFFFFFFFFFFFFFFll_ud1 0x7FFFFFFFFFFFFFFF_ud1LL 0x7FFFFFFFFFFFFFFFLL_ud1 
0x7FFFFFFFFFFFFFFE 0x7FFFFFFFFFFFFFFE 0x7FFFFFFFFFFFFFFEl 0x7FFFFFFFFFFFFFFEl 0x7FFFFFFFFFFFFFFEL 0x7FFFFFFFFFFFFFFEL 0x7FFFFFFFFFFFFFFEll 0x7FFFFFFFFFFFFFFEll 0x7FFFFFFFFFFFFFFELL 0x7FFFFFFFFFFFFFFELL 
0x7FFFFFFFFFFFFFFEu 0x7FFFFFFFFFFFFFFEu 0x7FFFFFFFFFFFFFFEul 0x7FFFFFFFFFFFFFFElu 0x7FFFFFFFFFFFFFFEuL 0x7FFFFFFFFFFFFFFELu 0x7FFFFFFFFFFFFFFEull 0x7FFFFFFFFFFFFFFEllu 0x7FFFFFFFFFFFFFFEuLL 0x7FFFFFFFFFFFFFFELLu 
0x7FFFFFFFFFFFFFFEU 0x7FFFFFFFFFFFFFFEU 0x7FFFFFFFFFFFFFFEUl 0x7FFFFFFFFFFFFFFElU 0x7FFFFFFFFFFFFFFEUL 0x7FFFFFFFFFFFFFFELU 0x7FFFFFFFFFFFFFFEUll 0x7FFFFFFFFFFFFFFEllU 0x7FFFFFFFFFFFFFFEULL 0x7FFFFFFFFFFFFFFELLU 
0x7FFFFFFFFFFFFFFE_ud1 0x7FFFFFFFFFFFFFFE_ud1 0x7FFFFFFFFFFFFFFE_ud1l 0x7FFFFFFFFFFFFFFEl_ud1 0x7FFFFFFFFFFFFFFE_ud1L 0x7FFFFFFFFFFFFFFEL_ud1 0x7FFFFFFFFFFFFFFE_ud1ll 0x7FFFFFFFFFFFFFFEll_ud1 0x7FFFFFFFFFFFFFFE_ud1LL 0x7FFFFFFFFFFFFFFELL_ud1 
0x8000000000000000 0x8000000000000000 0x8000000000000000l 0x8000000000000000l 0x8000000000000000L 0x8000000000000000L 0x8000000000000000ll 0x8000000000000000ll 0x8000000000000000LL 0x8000000000000000LL 
0x8000000000000000u 0x8000000000000000u 0x8000000000000000ul 0x8000000000000000lu 0x8000000000000000uL 0x8000000000000000Lu 0x8000000000000000ull 0x8000000000000000llu 0x8000000000000000uLL 0x8000000000000000LLu 
0x8000000000000000U 0x8000000000000000U 0x8000000000000000Ul 0x8000000000000000lU 0x8000000000000000UL 0x8000000000000000LU 0x8000000000000000Ull 0x8000000000000000llU 0x8000000000000000ULL 0x8000000000000000LLU 
0x8000000000000000_ud1 0x8000000000000000_ud1 0x8000000000000000_ud1l 0x8000000000000000l_ud1 0x8000000000000000_ud1L 0x8000000000000000L_ud1 0x8000000000000000_ud1ll 0x8000000000000000ll_ud1 0x8000000000000000_ud1LL 0x8000000000000000LL_ud1 
0xFFFFFFFFFFFFFFFF 0xFFFFFFFFFFFFFFFF 0xFFFFFFFFFFFFFFFFl 0xFFFFFFFFFFFFFFFFl 0xFFFFFFFFFFFFFFFFL 0xFFFFFFFFFFFFFFFFL 0xFFFFFFFFFFFFFFFFll 0xFFFFFFFFFFFFFFFFll 0xFFFFFFFFFFFFFFFFLL 0xFFFFFFFFFFFFFFFFLL 
0xFFFFFFFFFFFFFFFFu 0xFFFFFFFFFFFFFFFFu 0xFFFFFFFFFFFFFFFFul 0xFFFFFFFFFFFFFFFFlu 0xFFFFFFFFFFFFFFFFuL 0xFFFFFFFFFFFFFFFFLu 0xFFFFFFFFFFFFFFFFull 0xFFFFFFFFFFFFFFFFllu 0xFFFFFFFFFFFFFFFFuLL 0xFFFFFFFFFFFFFFFFLLu 
0xFFFFFFFFFFFFFFFFU 0xFFFFFFFFFFFFFFFFU 0xFFFFFFFFFFFFFFFFUl 0xFFFFFFFFFFFFFFFFlU 0xFFFFFFFFFFFFFFFFUL 0xFFFFFFFFFFFFFFFFLU 0xFFFFFFFFFFFFFFFFUll 0xFFFFFFFFFFFFFFFFllU 0xFFFFFFFFFFFFFFFFULL 0xFFFFFFFFFFFFFFFFLLU 
0xFFFFFFFFFFFFFFFF_ud1 0xFFFFFFFFFFFFFFFF_ud1 0xFFFFFFFFFFFFFFFF_ud1l 0xFFFFFFFFFFFFFFFFl_ud1 0xFFFFFFFFFFFFFFFF_ud1L 0xFFFFFFFFFFFFFFFFL_ud1 0xFFFFFFFFFFFFFFFF_ud1ll 0xFFFFFFFFFFFFFFFFll_ud1 0xFFFFFFFFFFFFFFFF_ud1LL 0xFFFFFFFFFFFFFFFFLL_ud1 
0xFFFFFFFFFFFFFFFE 0xFFFFFFFFFFFFFFFE 0xFFFFFFFFFFFFFFFEl 0xFFFFFFFFFFFFFFFEl 0xFFFFFFFFFFFFFFFEL 0xFFFFFFFFFFFFFFFEL 0xFFFFFFFFFFFFFFFEll 0xFFFFFFFFFFFFFFFEll 0xFFFFFFFFFFFFFFFELL 0xFFFFFFFFFFFFFFFELL 
0xFFFFFFFFFFFFFFFEu 0xFFFFFFFFFFFFFFFEu 0xFFFFFFFFFFFFFFFEul 0xFFFFFFFFFFFFFFFElu 0xFFFFFFFFFFFFFFFEuL 0xFFFFFFFFFFFFFFFELu 0xFFFFFFFFFFFFFFFEull 0xFFFFFFFFFFFFFFFEllu 0xFFFFFFFFFFFFFFFEuLL 0xFFFFFFFFFFFFFFFELLu 
0xFFFFFFFFFFFFFFFEU 0xFFFFFFFFFFFFFFFEU 0xFFFFFFFFFFFFFFFEUl 0xFFFFFFFFFFFFFFFElU 0xFFFFFFFFFFFFFFFEUL 0xFFFFFFFFFFFFFFFELU 0xFFFFFFFFFFFFFFFEUll 0xFFFFFFFFFFFFFFFEllU 0xFFFFFFFFFFFFFFFEULL 0xFFFFFFFFFFFFFFFELLU 
0xFFFFFFFFFFFFFFFE_ud1 0xFFFFFFFFFFFFFFFE_ud1 0xFFFFFFFFFFFFFFFE_ud1l 0xFFFFFFFFFFFFFFFEl_ud1 0xFFFFFFFFFFFFFFFE_ud1L 0xFFFFFFFFFFFFFFFEL_ud1 0xFFFFFFFFFFFFFFFE_ud1ll 0xFFFFFFFFFFFFFFFEll_ud1 0xFFFFFFFFFFFFFFFE_ud1LL 0xFFFFFFFFFFFFFFFELL_ud1 
0x10000000000000000 0x10000000000000000 0x10000000000000000l 0x10000000000000000l 0x10000000000000000L 0x10000000000000000L 0x10000000000000000ll 0x10000000000000000ll 0x10000000000000000LL 0x10000000000000000LL 
0x10000000000000000u 0x10000000000000000u 0x10000000000000000ul 0x10000000000000000lu 0x10000000000000000uL 0x10000000000000000Lu 0x10000000000000000ull 0x10000000000000000llu 0x10000000000000000uLL 0x10000000000000000LLu 
0x10000000000000000U 0x10000000000000000U 0x10000000000000000Ul 0x10000000000000000lU 0x10000000000000000UL 0x10000000000000000LU 0x10000000000000000Ull 0x10000000000000000llU 0x10000000000000000ULL 0x10000000000000000LLU 
0x10000000000000000_ud1 0x10000000000000000_ud1 0x10000000000000000_ud1l 0x10000000000000000l_ud1 0x10000000000000000_ud1L 0x10000000000000000L_ud1 0x10000000000000000_ud1ll 0x10000000000000000ll_ud1 0x10000000000000000_ud1LL 0x10000000000000000LL_ud1 
