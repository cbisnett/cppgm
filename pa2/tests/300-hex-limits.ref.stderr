ERROR: malformed number (#2): 0x7FFFFFFFl_ud1
ERROR: malformed number (#2): 0x7FFFFFFFL_ud1
ERROR: malformed number (#2): 0x7FFFFFFFll_ud1
ERROR: malformed number (#2): 0x7FFFFFFFLL_ud1
ERROR: malformed number (#2): 0x7FFFFFFEl_ud1
ERROR: malformed number (#2): 0x7FFFFFFEL_ud1
ERROR: malformed number (#2): 0x7FFFFFFEll_ud1
ERROR: malformed number (#2): 0x7FFFFFFELL_ud1
ERROR: malformed number (#2): 0x80000000l_ud1
ERROR: malformed number (#2): 0x80000000L_ud1
ERROR: malformed number (#2): 0x80000000ll_ud1
ERROR: malformed number (#2): 0x80000000LL_ud1
ERROR: malformed number (#2): 0xFFFFFFFFl_ud1
ERROR: malformed number (#2): 0xFFFFFFFFL_ud1
ERROR: malformed number (#2): 0xFFFFFFFFll_ud1
ERROR: malformed number (#2): 0xFFFFFFFFLL_ud1
ERROR: malformed number (#2): 0xFFFFFFFFl_ud1
ERROR: malformed number (#2): 0xFFFFFFFFL_ud1
ERROR: malformed number (#2): 0xFFFFFFFFll_ud1
ERROR: malformed number (#2): 0xFFFFFFFFLL_ud1
ERROR: malformed number (#2): 0x100000000l_ud1
ERROR: malformed number (#2): 0x100000000L_ud1
ERROR: malformed number (#2): 0x100000000ll_ud1
ERROR: malformed number (#2): 0x100000000LL_ud1
ERROR: malformed number (#2): 0x7FFFFFFFFFFFFFFFl_ud1
ERROR: malformed number (#2): 0x7FFFFFFFFFFFFFFFL_ud1
ERROR: malformed number (#2): 0x7FFFFFFFFFFFFFFFll_ud1
ERROR: malformed number (#2): 0x7FFFFFFFFFFFFFFFLL_ud1
ERROR: malformed number (#2): 0x7FFFFFFFFFFFFFFEl_ud1
ERROR: malformed number (#2): 0x7FFFFFFFFFFFFFFEL_ud1
ERROR: malformed number (#2): 0x7FFFFFFFFFFFFFFEll_ud1
ERROR: malformed number (#2): 0x7FFFFFFFFFFFFFFELL_ud1
ERROR: malformed number (#2): 0x8000000000000000l_ud1
ERROR: malformed number (#2): 0x8000000000000000L_ud1
ERROR: malformed number (#2): 0x8000000000000000ll_ud1
ERROR: malformed number (#2): 0x8000000000000000LL_ud1
ERROR: malformed number (#2): 0xFFFFFFFFFFFFFFFFl_ud1
ERROR: malformed number (#2): 0xFFFFFFFFFFFFFFFFL_ud1
ERROR: malformed number (#2): 0xFFFFFFFFFFFFFFFFll_ud1
ERROR: malformed number (#2): 0xFFFFFFFFFFFFFFFFLL_ud1
ERROR: malformed number (#2): 0xFFFFFFFFFFFFFFFEl_ud1
ERROR: malformed number (#2): 0xFFFFFFFFFFFFFFFEL_ud1
ERROR: malformed number (#2): 0xFFFFFFFFFFFFFFFEll_ud1
ERROR: malformed number (#2): 0xFFFFFFFFFFFFFFFELL_ud1
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: hex integer literal out of range (#1): 10000000000000000
ERROR: malformed number (#2): 0x10000000000000000l_ud1
ERROR: malformed number (#2): 0x10000000000000000L_ud1
ERROR: malformed number (#2): 0x10000000000000000ll_ud1
ERROR: malformed number (#2): 0x10000000000000000LL_ud1
